submissions/jonasMwansa/php/hobbyArrays.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+    //define and initialise the arrays
+    $firstnames = ['Jonas', 'Francis', 'James', 'Richard', 'Joseph', 'Vincent', 'Edwin', 'Aaron'];
+    $hobbies = ['programming', 'playing football', 'betting', 'laboratory experiments', 'playing guitar', 'programming', 'photography', 'exploration'];
+
+    //we loop through
+    foreach( $firstnames as $hobby => $name ) {
```

Comment:
$firstnames as $hobby => $name?

Reply:
alright thank you @ToluEnny24 , let me just do it the easier way